docs/conf.py

```diff
@@ -63,8 +63,41 @@
     "myst_nb",
     "sphinx_design",
     "sphinx_inline_tabs",
+    # Inheritance diagrams - possible remove graphviz and inheritance
+    "sphinx.ext.graphviz",
+    "sphinx.ext.inheritance_diagram",
+    "sphinxcontrib.mermaid",
 ]
 
+# Inheritance diagram settings - you can find various settings in the
+# graphviz docs here https://graphviz.org/doc/info/attrs.html
+inheritance_graph_attrs = dict(
+    rankdir="BT",
+    color="darkorchid4",
+    size='"6.0, 8.0"',
+    fontsize=16,
+    ratio="compress",
+)
+
+inheritance_node_attrs = dict(
+    # https://graphviz.org/docs/attrs/rankdir/
+    # rankdir="BT",
+    shape="ellipse",
```

Comment:
Please nitpick away. Jonatan I've never worked with UML before so i'm learning here. 

I"ll work on this now

And in the meantime i'd like to share with you what Chat GPT provided to me when i asked it to generate UML diagrams twice for stravalib. 😆 

Do these look UML compliant to you? 🙃 chatgpt was sure they were and it was sure it know all about stravalib's API. 

<img width="959" alt="Screenshot 2024-09-15 at 3 35 30 PM" src="https://github.com/user-attachments/assets/67092d03-be69-48c7-9cce-5187cd003918">


<img width="950" alt="Screenshot 2024-09-15 at 3 35 14 PM" src="https://github.com/user-attachments/assets/09f97d0f-0869-4866-973c-9ea1378f4c0a">

Reply:
Wow, spectacular hallucinations 🤪. More than 20 years ago, we already had tools that could easily reverse-engineer UML diagrams from code.

Reply:
so true! Ihave started a running experiment that when i work on things, i ask ChatGPT here and there to see what it returns. i was sure it could do a better job with a basic diagram. But NOPE!